spec/models/setup_spec.rb

```diff
@@ -0,0 +1,59 @@
+require_relative '../rails_helper'
+
+describe Setup do
+  let(:params) { ActionController::Parameters.new({
```

Comment:
Think about moving this into a factory using FactoryGirl per this example here: (https://github.com/churchio/onebody/blob/master/spec/factories/prayer_requests.rb)